src/olympia/users/models.py

```diff
@@ -642,6 +642,8 @@ def fetch_names():
 RESTRICTION_TYPES = Choices(
     ('SUBMISSION', 1, 'Submission'),
     ('APPROVAL', 2, 'Approval'),
```

Comment:
Maybe we should rename these to include `addon` in there for clarity.

Reply:
https://github.com/mozilla/addons-server/pull/20581/commits/ac9b4f704e9c1a88e7b14dc2815325833a99ffab